rclpy/test/test_waitable.py

```diff
@@ -175,7 +175,12 @@ def __init__(self, node):
 
         with node.handle:
             self.subscription = _rclpy.Subscription(
-                node.handle, EmptyMsg, 'test_topic', QoSProfile(depth=10).get_c_qos_profile())
+                node.handle,
+                EmptyMsg,
+                'test_topic',
+                QoSProfile(depth=10).get_c_qos_profile(),
+                None
```

Comment:
I believe that we need to add more test cases here,

- actually use `SubscriptionOptions` as argument. (Not `None` case)
- what is the default value for `SubscriptionOptions`?
- verify `SubscriptionOptions` member variables set by user application?

Reply:
Good idea. I'll add a test case for this.